module/stylesheets/components/_divider.scss

```diff
@@ -0,0 +1,16 @@
+
+.divider {
+    border-top: 1px solid;
+  border-color: #333;
```

Comment:
Fix Indentation and change these hard coded values to variables.

Reply:
Done